test/test_matcher.py

```diff
@@ -67,309 +66,257 @@ def match(data):
         raise yara.Error()
 
 
-def get_records(bucket, key, num=1):
+def get_records(num=1):
```

Comment:
This needs refactoring now so that you get `num` array elements in the `Records` array with each record having a single  body object.